packages/cli/src/ceramic-daemon.ts

```diff
@@ -177,8 +177,8 @@ class CeramicDaemon {
    * @dev Useful when the docId is unknown, but you have the genesis contents
    */
   async createDocFromGenesis (req: Request, res: Response): Promise<void> {
-    const { doctype, genesis, docOpts } = req.body
-    const doc = await this.ceramic.createDocumentFromGenesis(doctype, DoctypeUtils.deserializeCommit(genesis), docOpts)
+    const { doctype, genesis, opts } = req.body
```

Comment:
This is technically a breaking change to the HTTP api, but a minor one.  It seemed worth doing to avoid having an arg called `docOpts` when we're moving away from calling all doctypes (streamtypes) documents.

Reply:
Pulled this change out to its own PR: https://github.com/ceramicnetwork/js-ceramic/pull/1227